src/Core/References/NugetPackages.cs

```diff
@@ -175,32 +177,34 @@ public async Task<PackageIdentity> ParsePackageId(string package)
         /// <summary>
         /// Adds a new package given the name and version as strings.
         /// </summary>
-        public async Task<PackageIdentity> Add(string package)
+        public async Task<PackageIdentity> Add(string package, Action<string>? statusAction = null)
```

Comment:
Maybe call this argument `statusCallback`?

Reply:
Good idea, I'll go on and change accordingly.